shared/contracts/BaseTemplate.sol

```diff
@@ -152,21 +150,26 @@ contract BaseTemplate is APMNamehash, IsContract {
 
     /* VOTING */
 
+    function _installVotingApp(Kernel _dao, MiniMeToken _token, uint64[3] _votingSettings) internal returns (Voting) {
```

Comment:
```suggestion
    function _installVotingApp(Kernel _dao, MiniMeToken _token, uint64[3] memory _votingSettings) internal returns (Voting) {
```

Just to be a bit more explicit :)

Reply:
I'd change all the arrays passed by argument in that case